src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/ServiceBusTriggerBinding.cs

```diff
@@ -94,7 +95,8 @@ public string EntityPath
         public async Task<ITriggerData> BindAsync(object value, ValueBindingContext context)
         {
             BrokeredMessage message = value as BrokeredMessage;
-            if (message == null && !_converter.TryConvert(value, out message))
+            if (message == null && _converter != null && 
+                !_converter.TryConvert(value, out message))
```

Comment:
This if block is about trying to do a conversion. We only want to do that if the message isn't already a BrokeredMessage, and we actually have a converter. I just put this here to be defensive, since this was the source of the null ref. converter can't actually be null anymore